packages/facteur/facteur.0.1.0/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+license:      "MIT"
+authors:      [ "Romain Calascibetta <romain.calascibetta@gmail.com>" ]
+maintainer:   [ "Romain Calascibetta <romain.calascibetta@gmail.com>" ]
+homepage:     "https://github.com/dinosaure/facteur"
+bug-reports:  "https://github.com/dinosaure/facteur/issues"
+dev-repo:     "git+https://github.com/dinosaure/facteur.git"
+doc:          "https://dinosaure.github.io/facteur/"
+synopsis:     "Tool to send an email"
+description: """Tool to send an email in OCaml according RFC4409 [SUBMIT]
+"""
+
+build: [ "dune" "build" "-p" name "-j" jobs ]
+run-test: [ "dune" "runtest" "-p" name "-j" jobs ]
+
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "mrmime"
+  "colombe"
+  "sendmail-lwt"
+  "logs"
+  "fmt"
+  "astring"
+  "fpath"
+  "cmdliner"
+  "domain-name" {>= "0.3.0"}
+  "x509" {>= "0.7.0"}
+  "emile" {>= "0.5"}
+  "dune" {>= "1.3"}
+]
+
+depexts: [
```

Comment:
Would it be possible to have those depexts be defined in their own package? I'm not too sure if the depexts are well defined but if it's possible it would be better off in a meta package in my opinion.

Reply:
Currently, this depext comes from the `magic` package which uses `libmagic-dev` too. About a meta-package, I don't know well how to do that.